src/main/webapp/index.html

```diff
@@ -6,22 +6,46 @@
 <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
 
 <div id="app">
-    <search-page></search-page>
+  <v-app>
+    <navbar></navbar>
+    <!-- route outlet -->
+    <!-- component matched by the route will render here -->
+    <router-view></router-view>
+  </v-app>
 </div>
 
-<!-- Vue -->
-<script src="https://cdn.jsdelivr.net/npm/vue@2.x/dist/vue.js"></script> <!-- TODO: This is dev Version, switch to prod for deployment -->
-<!-- HTTP Vue Loader -->
+<!-- Vue --> <!-- TODO: This is dev Version, switch to prod for deployment -->
+<script src="https://cdn.jsdelivr.net/npm/vue@2.x/dist/vue.js"></script>
+<!-- Vue Router --> <!-- TODO: This is dev Version, switch to prod for deployment -->
+<script src="https://unpkg.com/vue-router/dist/vue-router.js"></script>
+<!-- HTTP Vue Loader --> <!-- TODO: This is dev Version, switch to prod for deployment -->
 <script src="https://unpkg.com/http-vue-loader"></script>
 <!-- Vuetify -->
 <script src="https://cdn.jsdelivr.net/npm/vuetify@2.x/dist/vuetify.js"></script>
 <!-- Declares the Vue instance-->
 <script>
-  new Vue({
-    el: '#app',
-    vuetify: new Vuetify(),
+  // 1. Define route components.
+  const Home = httpVueLoader('/views/HomePage.vue')
+  const Search = httpVueLoader('/views/SearchPage.vue')
+  const Profile = httpVueLoader('/views/ProfilePage.vue')
+
+  // 2. Define some routes
+  const routes = [
+    { path: '/', component: Home },
+    { path: '/search', component: Search }
+  ]
+
+  // 3. Create the router instance and pass the `routes` option
+  const router = new VueRouter({
+    routes // short for `routes: routes`
```

Comment:
nit: if you're likely not going to add anything to this object, especially since it's just a shorthand, I think you can include this inline above

Reply:
Makes sense, I can make that work :)